std/algorithm/searching.d

```diff
@@ -3170,15 +3264,15 @@ Returns: The maximal element of the passed-in range.
 See_Also:
     $(XREF algorithm, comparison, max)
 */
-auto maxElement(alias map = "a", Range)(Range r)
+auto maxElement(alias map = IdentityFun, Range)(Range r)
     if (isInputRange!Range && !isInfinite!Range &&
         !is(CommonType!(ElementType!Range, RangeElementType) == void))
 {
     return extremum!(map, "a > b")(r);
 }
 
 /// ditto
-auto maxElement(alias map = "a", Range, RangeElementType = ElementType!Range)
+auto maxElement(alias map = IdentityFun, Range, RangeElementType = ElementType!Range)
```

Comment:
As I said in another PR, private templates should never show up in public template constraints.